Automattic/wc-calypso-bridge
# Add parameter to go back for theme links in cys intro screen

### Changes proposed in this Pull Request:

Partially addresses https://github.com/woocommerce/woocommerce/issues/41230



### How to test the changes in this Pull Request:

1. Set up a latest WordPress according to this guide: p6q8Tx-3Je-p2
1. Make sure to enable `customize-store` feature flag
2. Go to `Home` and click on `Customize your store` task
3. In the intro screen, click on any of the themes
4. Observe the URL navigated has `from=customize-store`. For example, `/theme/tsubaki/wooexpresstestingsite.wpcomstaging.com?from=customize-store`




### Other information:




### FOR PR REVIEWER ONLY:
